src/Datadog.Trace.ClrProfiler.Native/cor_profiler.cpp

```diff
@@ -161,11 +161,21 @@ CorProfiler::Initialize(IUnknown* cor_profiler_info_unknown) {
          "Please ensure that there is only one AppDomain or, if applications are being hosted in IIS, ",
          "ensure that all Application Pools have at most one application each. ",
          "Otherwise, a sharing violation (HRESULT 0x80131401) may occur.");
+    Info("Note: This flag is unnecessary when running against the .NET Framework 4.5.2 or higher.");
```

Comment:
Since we check for `ICorProfilerInfo6` (net452+) a few lines below, changing this:
- in net45 and net451, log the original message above
- in net452+, "this env var is  not needed in net452 and above and will be ignored"

Reply:
Good idea. Fixed in 15bbeaad2bd94c2cef1e72503cecf1f1174fecc6